analysis/classification/utils/db_functions.py

```diff
@@ -152,7 +152,7 @@ def display_labelling_progress(token='admin'):
 _TO_LABEL_REFRESH = 100
 
 
-def _pull_thread(cv, data_to_label, bq_client, user_langs, method):
+def _pull_thread(cv, data_to_label, bq_client, user_langs, method, target_probability=0.5):
```

Comment:
As I previously did for `method` please pass encapsulated in a list.  This is as lists are mutable and so if the value changes while the thread is running, it will pick up the new value.  Or feel free to use another solution to ensure this if you prefer.